src/Yoti/ActivityDetails.php

```diff
@@ -113,4 +115,152 @@ public function getUserId()
     {
         return $this->_rememberMeId;
     }
+
+    /**
+     * @return null|string
+     */
+    public function getFamilyName()
+    {
+        return $this->getProfileAttribute(self::ATTR_FAMILY_NAME);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getGivenNames()
+    {
+        return $this->getProfileAttribute(self::ATTR_GIVEN_NAMES);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getFullName()
+    {
+        return $this->getProfileAttribute(self::ATTR_FULL_NAME);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getDateOfBirth()
+    {
+        return $this->getProfileAttribute(self::ATTR_DATE_OF_BIRTH);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getGender()
+    {
+        return $this->getProfileAttribute(self::ATTR_GENDER);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getNationality()
+    {
+        return $this->getProfileAttribute(self::ATTR_NATIONALITY);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPhoneNumber()
+    {
+        return $this->getProfileAttribute(self::ATTR_PHONE_NUMBER);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getSelfie()
+    {
+        return $this->getProfileAttribute(self::ATTR_SELFIE);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getEmailAddress()
+    {
+        return $this->getProfileAttribute(self::ATTR_EMAIL_ADDRESS);
+    }
+
+    /**
+     * @return null|string
+     */
+    public function getPostalAddress()
+    {
+        return $this->getProfileAttribute(self::ATTR_POSTAL_ADDRESS);
+    }
+
+    /**
+     * Gets a user generic profile attribute
+     * @param $param
+     * @return mixed|null
+     */
+    protected function get($param)
+    {
+        // Getting attribute $param
+        return (!empty($param)) ? $this->getProfileAttribute($param) : null;
+    }
+
+    /**
+     * Check the number of arguments supplied
+     * @param array $args
+     * @param int $min
+     * @param int $max
+     * @param $method
+     */
+    protected function checkArguments(array $args, $min, $max, $method)
+    {
+        $argCounts  = count($args);
+        $min        = (int)$min;
+        $max        = (int)$max;
+
+        if($argCounts < $min || $argCounts > $max) {
+            throw new \Exception("Method {$method} needs minimum {$min} and maximum {$max} argument(s), {$argCounts} arguments given");
+        }
+    }
+
+    /**
+     * Handles a call to an undefined function
+     * @param $method
+     * @param $args
+     * @return mixed|null|void
+     */
+    public function __call($method, $args)
+    {
+        // Check method name is alphanumeric
+        if (!preg_match('~^[0-9a-z]+$~i', $method)) {
```

Comment:
Not sure what the conventions are, but I'm pretty sure you should allow underscores as well.